package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>gz-cmake3</name>
+  <version>3.5.0</version>
+  <description>Gazebo CMake : CMake Modules for Gazebo Projects</description>
+
+  <maintainer email="scpeters@openrobotics.org">Steve Peters</maintainer>
+
+  <license>Apache License 2.0</license>
+
+  <url type="website">https://github.com/gazebosim/gz-cmake</url>
+
+  <buildtool_depend>cmake</buildtool_depend>
```

Comment:
```suggestion
  <depend>cmake</depend>
```
Probably not a big deal but we can declare the `cmake` dependency as build-time, exec-time and build-export-depend dependency since we probably expect `cmake` to be available in all these scenarios.

pkg-config could fit into exec-time but not in all scenarios probably. Good as it is for me.

Reply:
Is the question whether we expect `cmake` to be available or that we only need it at build time? I was under the impression that we want to narrow down the scope as much as possible.

Reply:
I think we don't use `cmake` when executing gazebo through any of the `gz` commands, but we do invoke `cmake` in some of our tests; is that a valid reason for the difference between `buildtool_depend` and `depend`?

Reply:
I think `<buildtool_depend>cmake</buildtool_depend>` is the right thing here. I have looked at a few examples:

- https://github.com/eclipse-iceoryx/iceoryx/blob/e274957ac619cd9881687e688e3f8d1c28ff115c/iceoryx_binding_c/package.xml#L13
- https://github.com/eProsima/Fast-DDS/blob/ee0637d7a3302a1c69377a0282da7b0f7096292a/package.xml#L25
- https://github.com/ros-industrial/stomp/blob/f9e9d78f741f0779b65333d4f54d68ade740520b/package.xml#L14

It is also what's recommended in https://ros.org/reps/rep-0136.html

> I think we don't use cmake when executing gazebo through any of the gz commands, but we do invoke cmake in some of our tests; is that a valid reason for the difference between buildtool_depend and depend?

Ah, perhaps we'll need `<test_depend>cmake</test_depend>` to cover that.

Reply:
I've added `<test_depend>cmake</test_depend>` in https://github.com/gazebosim/gz-cmake/pull/413/commits/0f91fadf5dde649b332d601e6fc18895d2ef2a7a